pkg/osbuild/bootupd_stage_test.go

```diff
@@ -143,3 +145,107 @@ func TestBootupdStageJsonHappy(t *testing.T) {
   ]
 }`)
 }
+
+func TestGenBootupdDevicesMountsMissingRoot(t *testing.T) {
+	filename := "fake-disk.img"
+	pt := &disk.PartitionTable{}
+	_, _, err := osbuild.GenBootupdDevicesMounts(filename, pt)
+	assert.EqualError(t, err, "required mounts for bootupd stage [/ /boot /boot/efi] missing")
+}
+
+const (
+	MebiByte = 1024 * 1024        // MiB
```

Comment:
Can we take these constants from somewhere else? I believe we have them either in `disk`, or `common`.

Reply:
Thanks, yes!